week3/assign1/src/components/Title.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+const Title =({title})=>{
+  return(
+    <>
```

Comment:
```suggestion
     return (
     <TitleSec>
        <p>{title}</p>
    </TitleSec>
    )
```
나는 필요없는 fragment는 되도록 안쓰는편이얌!